docs/proposals/excluding-drives.md

```diff
@@ -0,0 +1,105 @@
+# Proposal: Specific drives excluding
+
+Last updated: 07.02.2022
+
+
+## Abstract
+
+CSI Baremetal needs to have API for excluding specific drives from the common storage pool. 
+Excluded drives won't be used for allocating new Volumes or LVGs.
+
+## Background
+
+User stories:
+1. Deploy another CSI driver in kube-cluster.
+2. Reserve space for increasing root partition.
```

Comment:
Not sure what does it mean...

Reply:
To have one free drive in the system for a critical situation. Is it not applicable?